File: CursoPython3/exer/lista64Exer/exer32.py
"""
32) Escreva um programa que calcule e exiba a soma dos quadrados dos 20 primeiros números inteiros positivos 
ímpares a partir do número informado pelo usuário menor que 10 e maior que zero. 
"""

number = int(input("Digite um número entre 1 e 9: "))

while number <= 0 or number >= 10:
    number = int(input("Digite um número entre 1 e 9: "))

sum_of_squares = 0

for i in range(number, number + 40, 2):
    sum_of_squares += i**2

print(
    f"A soma dos quadrados dos 20 primeiros números ímpares a partir de {number} é {sum_of_squares}")

File: CursoJava/Exer/src/Outros/lista64Exer/Exer32.java

/*******************************************************************************
* 32) Escreva um programa que calcule e exiba a soma dos quadrados dos 20 primeiros
* números inteiros positivos ímpares a partir do número informado pelo usuário menor que 10
* e maior que zero.
*******************************************************************************/

import java.util.Scanner;

public class Exer32 {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		System.out.print("Enter a positive integer less than 10 and greater than zero: ");
		int number = sc.nextInt();

		int sumSquares = 0;
		int countOdd = 0;

		for (int i = number; countOdd < 20; i++) {
			if (i % 2 == 1) {
				sumSquares += i * i;
				countOdd++;
			}
		}

		System.out.println(
				"Sum of squares of the first 20 odd positive integers starting from " + number + ": " + sumSquares);

		sc.close();
	}

}
